### PR TITLE
Moved hardcoded version, releasedate in MainController.js to release.json

### PR DESCRIPTION
**Pull request by terencemo**
Removed hard-coded Mifos X version and release date from MainController.js, using json config file instead

### CODE REVIEW

**Comment by terencemo** [comment]
@Nayan, @vishwasbabu review this please

File: app/VERSION.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+1.26.0
```
Comment (vishwasbabu) [inline app/VERSION.txt]: I guess this isn't being used ?

**Comment by vishwasbabu** [comment]
Looks perfect @terencemo :+1:

File: app/VERSION.txt (hunk 1 above)
Comment (terencemo) [inline app/VERSION.txt]: @vishwasbabu sorry for the late update. Yes, VERSION.txt was replaced by release.json but the file was not removed. Will do a separate change for this.

**Pull request merged**